pkgs/os-specific/linux/cfs-zen-tweaks/default.nix

```diff
@@ -0,0 +1,46 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, makeWrapper
+, gawk
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cfs-zen-tweaks";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "igo95862";
+    repo = "cfs-zen-tweaks";
+    rev = version;
+    sha256 = "HRR2tdjNmWyrpbcMlihSdb/7g/tHma3YyXogQpRCVyo=";
+  };
+
+  postPatch = ''
+    patchShebangs .
+  '';
+
+  postInstall = ''
+    chmod +x $out/lib/cfs-zen-tweaks/set-cfs-zen-tweaks.bash
+    wrapProgram $out/lib/cfs-zen-tweaks/set-cfs-zen-tweaks.bash \
+      --prefix PATH : ${gawk}/bin
```

Comment:
It is only used here https://github.com/igo95862/cfs-zen-tweaks/blob/93b8cbac971d9d6cb1038716d05053f85f5d6e9f/set-cfs-zen-tweaks.bash#L36 so we can also patch it which is usually nicer than wrapping.